src/main/java/net/sf/jabref/gui/autocompleter/AutoCompleteSupport.java

```diff
@@ -233,9 +234,9 @@ private void postProcessTextChange() {
         }
 
         String text = textComp.getText();
-        E[] candidates = autoCompleter.complete(text);
+        List<E> candidates = autoCompleter.complete(text);
         renderer.update(candidates);
-        if (textComp.isEnabled() && (candidates != null) && (candidates.length > 0)) {
+        if (textComp.isEnabled() && (candidates != null) && (candidates.size() > 0)) {
```

Comment:
Yes, one should never return null - which is true for lists as well. ;)

Please remove the null check, as it is obsolete.